src/frontend/src/flows/loginUnknown.ts

```diff
@@ -39,19 +39,19 @@ const pageContent = (userIntent: UserIntent) => html` <style>
     ${icLogo}
     <h2 id="loginWelcome">Welcome to<br />Internet Identity</h2>
     <p>
-      Provide an identity anchor to
+      Provide an Identity Anchor to
       authenticate${authenticateUnknownIntent(userIntent)}.
     </p>
     <input
       type="text"
       id="registerUserNumber"
-      placeholder="Enter Identity anchor"
+      placeholder="Enter Identity Anchor"
     />
     <button type="button" id="loginButton" class="primary">Authenticate</button>
     ${userIntent.kind === "addDevice"
       ? html`<div class="spacer"></div>`
       : html`<div class="textLink" id="registerSection">
-            New user?
+            New?
```

Comment:
Just because this seems to be an unrelated change: Do you actually want to change "New user" to "New"?

Reply:
Good spot. Jan asked me to make that change so I added it but it doesn't really fit the topic of this PR. Would you prefer I move it to a later terminology PR?

Reply:
I'm completely fine with leaving it in this PR, just asked to make sure that this change was intentional!

Reply:
Great! Understood.